index.html

```diff
@@ -212,8 +215,8 @@ <h2>
       <p>
         The term <dfn>DOM</dfn> is used to refer to the API set made available
         to scripts in Web applications, and does not necessarily imply the
-        existence of an actual <a data-cite="DOM#document">Document</a> object
-        or of any other <a data-cite="DOM#node">Node</a> objects as defined in
+        existence of an actual [=Document=]</a> object
+        or of any other [=Node=]</a> objects as defined in
```

Comment:
Late to the party here, but couple of stray closing `</a>`, and below. Otherwise, looking good.

Reply:
yeah, we found out when tidy borked, so removed them in a followup PR